.github/workflows/ci_eval.yaml

```diff
@@ -58,5 +59,51 @@ jobs:
             -e "git+https://github.com/iree-org/iree-turbine.git#egg=iree-turbine"
           pip install --no-compile -r requirements.txt -r sharktank/requirements-tests.txt -e sharktank/
 
-      - name: Run perplexity test
-        run:  pytest -n 4 -v -s sharktank/tests/evaluate/perplexity_test.py  --longrun
+      - name: Run perplexity test with vmfb
+        run:  pytest -n 4 -v -s sharktank/tests/evaluate/perplexity_vmfb_test.py --iree-device='hip://7' --longrun
```

Comment:
Why -n 4? This is running on a system with something like 192 cores so we could expand that quite a bit to maybe speed runtime up.